src/main/java/card/validation/ValidateCard.java

```diff
@@ -0,0 +1,103 @@
+package main.java.card.validation;
+
+import main.java.card.issuer.CardIssuerParameters;
+import main.java.card.information.CreditCardInformation;
+import main.java.card.issuer.CardIssuers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class ValidateCard {
+
+    public enum ResultTypeEnum {
+        SUCCESS("Validated"),
+        ERR_ISUER("Incorrect issuer name"),
+        ERR_LENGTH("Incorrect card length"),
+        ERR_LUHN("Card number didn't successfully pass Luhn validation");
+
+        ResultTypeEnum(String msg){
+            message = msg;
+        }
+        public String getMessage() {
+            return message;
+        }
+
+        private String message;
+    }
+
+
+    public static ResultTypeEnum performValidation(String[] inputArgs){
+        CardInputArgumentsControl inputControl = new CardInputArgumentsControl(inputArgs, 2);
+
+        try {
+            inputControl.checkInput();
+        }
+        catch(IOException e){
+            System.out.println(e);
+            System.exit(1);
+        }
+
+        CreditCardInformation creditCard = new CreditCardInformation(inputArgs);
+
+        if(!CardIssuers.doesIssuerExist(creditCard.getCardIssuerName()))
+            return ResultTypeEnum.ERR_ISUER;
+
+        if (!checkCardNumberLength(creditCard))
+            return ResultTypeEnum.ERR_LENGTH;
+
+        if (!validateCardNumber(creditCard.getCardNumber()))
+            return ResultTypeEnum.ERR_LUHN;
+
+        return ResultTypeEnum.SUCCESS;
+    }
+
+    private static boolean checkCardNumberLength(CreditCardInformation creditCard){
+        String currentCardIssuerName = creditCard.getCardIssuerName();
+        String currentCardNumber = creditCard.getCardNumber();
+        int currentCardNumberLength = currentCardNumber.length();
+
+        CardIssuerParameters cardIssuer =  CardIssuers.getIssuerParameters(currentCardIssuerName);
+        ArrayList<Integer> lengthsOfCardNumber = cardIssuer.getLengthOfCardNumber();
+
+        for(Integer numberLength : lengthsOfCardNumber)
+            if (numberLength == currentCardNumberLength)
+                return true;
+
+        return false;
+    }
+
+    private static boolean validateCardNumber(String cardNumber){
```

Comment:
Rename method `isValidByLuhnAlgorithm` may be more clear.
Remove comment from code.  `//it uses Luhn algorithm`
Consider taking this method to a separate class `LuhnAlgorithmValidator`. you can also use an interface with one method `isValid()`.
```suggestion
    private static boolean isValidByLuhnAlgorithm(String cardNumber){
```